src/validatePrTitle.js

```diff
@@ -54,11 +58,13 @@ module.exports = async function validatePrTitle(
     );
   }
 
-  if (scopes && result.scope && !scopes.includes(result.scope)) {
+  const givenScopes = result.scope ? result.scope.split(',') : undefined;
+  const unknownScopes = givenScopes ? givenScopes.filter(isUnknownScope) : [];
+  if (scopes && unknownScopes.length > 0) {
     throw new Error(
-      `Unknown scope "${
-        result.scope
-      }" found in pull request title "${prTitle}". Use one of the available scopes: ${scopes.join(
+      `Unknown scope(s) "${unknownScopes.join(
```

Comment:
Let's handle the plural for slightly better error messages:

```suggestion
      `Unknown ${unknownScopes.length > 1 ? 'scopes' :'scope'} "${unknownScopes.join(
```

Reply:
👍